CHANGELOG.md

```diff
@@ -11,6 +11,8 @@
 - Remove `Sql.get_conn()` interface in favor of `.call()` and `.transaction()`. #4055
 - Updated provider database.
 - Disable DKIM-Checks again #4076
+- Remove metadata from avatars and JPEG images before sending #4037
```

Comment:
does this mean that "small" PNG that don't get recoded don't get  its metadata removed?

Reply:
Yes, currently. But i'll try to remove Exif as wrote above